content/en/docs/reference/command-line-tools-reference/feature-gates/watch-list.md

```diff
@@ -6,8 +6,8 @@ _build:
   render: false
 
 stages:
-  - stage: alpha 
-    defaultValue: false
-    fromVersion: "1.27"
+  - stage: beta
+    defaultValue: true
+    fromVersion: "1.31"
```

Comment:
@p0lyn0mial The proposed change doesn’t seem accurate. As the feature transitions into the beta phase, we need to add a new stage entry within the stages block labeled ‘beta’, rather than modifying the existing alpha stage to beta. Please see the suggestion below and refer to the documentation [here](https://kubernetes.io/docs/contribute/new-content/new-features/#ready-for-review-feature-gates) for further information.

```suggestion
  - stage: alpha 
    defaultValue: false
    fromVersion: "1.27"
    toVersion: "1.30"    
  - stage: beta
    defaultValue: true
    fromVersion: "1.31"
```